### ISSUE TITLE
The deadzone feature isn't working anymore for my PS4 controllers (edited for formatting)

### ISSUE THREAD

**Issue by MaximusFullius** [Ryochan7/DS4Windows#1897]
I'm using a PS4 controller connected to my PC via wireless USB dongle, and I've noticed in the profile settings that the deadzone settings are no longer effecting the output variables in LX (O), LY (O), RX (O) or RY (O). This wasn't happening in previous updates, but it seems like some update broke it. I've been experiencing this problem for a few months now, and I finally decided to submit a report about it. 

I've uploaded a picture of what this bug looks like in the program: In the "Controller Readings" tab of the profile settings, you will see output mappings of the left and right sticks next to the SixAxis output. When you change the deadzone settings in the Axis Config tab of the profile editor, it creates a red circle on the Left or Right output maps to show how large you set your deadzone to be. The way deadzone is supposed to work is that while the stick is kept within this deadzone, the output is supposed to be zero deviation from the neutral center (i.e: the X and Y outputs should stay in the middle at 128 until the deadzone is surpassed). The bug I'm experiencing is that the output values aren't staying in the middle when I move the stick, so the deadzone isn't keeping the output values in the middle. I've increased the left analogue stick's deadzone massively just to show the problem. I don't normally play with the deadzone this big, of course, but I do need to use _some_ deadzone because my analogue sticks's outputs twitch-around a little when they are in neutral position. This deadzone bug occurs with both sticks and even different PS4 controllers. In this screenshot, the output values are supposed to stay in the middle "128" value until the deadzone is surpassed, but as you can see with the left analogue stick's output values, the numbers are changing with all stick movements within the deadzone, which indicates that the deadzone isn't working. 
 
![image](https://user-images.githubusercontent.com/12753462/108895938-db263780-75d9-11eb-9861-0473fad3bf21.png)

**To Reproduce:**
1. Create a new profile in DS4Windows
2. Try setting the deadzone in the Axis Config.
3. Notice how when you move the analogue stick with the set deadzone, the output in the "Controller Readings" window doesn't stay the center value within the deadzone.

Expected Behavior: The output values stay "128" while the input is within the deadzone.

**Desktop:**
Controller Make and Model: The sticker on the back of my controller has come off, so I don't have the exact model number anymore. If it matters, I got this in the US.
OS: Windows 10 Home 20H2
DS4Windows Version: The latest version

**Comment by Ryochan7** [comment]
Works for me TM

**Comment by Ryusennin** [comment]
I've just tested some dead zone / anti dead zone solutions for Mirror's Edge Catalyst, and it works perfectly.

**Comment by MaximusFullius** [comment]
So I just figured something out: In games, the deadzone values work, but in the DS4Windows program, the output values shown in the "Controller Readings" tab don't indicate that since the output values are shown as the same as the input values when within the deadzones. As long as the deadzone actually works in-games, I suppose this in-program bug doesn't matter, but I was confused when I saw this. I should have tested the deadzone in-games. Thanks for the replies.

> 
> 
> Works for me TM
